app/views/results/_defendant_court_applications.html.haml

```diff
@@ -0,0 +1,3 @@
+- results.each do |defendant_court_applications|
+  -# = link_to defendant_court_applications.title, court_application_path(defendant_court_applications.id), class: 'govuk-link'
+  = link_to defendant_court_applications.title, "#{'#' + defendant_court_applications.id}", class: 'govuk-link'
```

Comment:
Just eyeballing it, but it looks to me like we'd probably want to add a little margin-bottom here, otherwise it's hard to visually distinguish two appeals as being distinct links:
<img width="656" alt="Screenshot 2025-04-15 at 15 04 40" src="https://github.com/user-attachments/assets/687f9ecf-5f3f-4407-936f-c2e0a33ef4fc" />

Reply:
I'll add the margin this will change if you have more than two application it does not work.